### PR TITLE
Help Command and update GUI

### LINKED ISSUE #17: Clean & Presentable GUI: As a minimalist student, I want a clean interface that is neatly laid out so I won't be overwhelmed by content

### LINKED ISSUE #26: help command: As a forgetful user, I want to have help options for the application, so that I can use it when I forget commands.

### LINKED ISSUE #45: GUI and Handling user Input - display a help guide to Calgo's available commands via GUI

### PR DESCRIPTION
**Pull request by J-Dan23**
Help Command:
Change help command to contain (summarised) UG in pop-up window
Change help window GUI to display (summarised) UG

General:
Change references to Address Book in app name bar, initialisation, to show Calgo instead
Change app logo to Calgo logo (note image name is still the initial name)

### CODE REVIEW

**Comment by thrivinb** [comment]
LGTM

**Comment by jeremylow97** [comment]
Looks good! :) I sincerely love it and am proud of your work

**Comment by eugenetyc** [comment]
Looks good to me. Thanks for the good work!

**Review by thrivinb — APPROVED** [review]
LGTM

**Review by jeremylow97 — APPROVED** [review]
I reviewed and things look okay. Good job am proud

File: src/main/java/f11_1/calgo/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,8 +16,67 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
+    public static final String HELP_UPDATE_MESSAGE = "update:\n"
```
Comment (eugenetyc) [inline src/main/java/f11_1/calgo/ui/HelpWindow.java:19]: Looks good to me. Thanks for the good work!

**Review by eugenetyc — COMMENTED** [review]
Looks good to me. Thanks for the good work!

**Pull request merged**